pkgs/top-level/python-packages.nix

```diff
@@ -15341,6 +15402,25 @@ in {
     };
   };
 
+  graphviz = buildPythonPackage rec {
+    name = "graphviz-${version}";
+    version = "0.5.2";
+    disable = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/01/98/8dec899491e4ac01a6fc8269e3b6dffd35421321b6858e21672489678fa8/graphviz-0.5.2.zip";
```

Comment:
You could replace this with:

```
url = "mirror://pypi/g/graphviz/${name}.zip";
```
This makes updating the derivation easier singe changing the `version` attribute is enough to update the url as well.